pallets/phala/src/stakepool.rs

```diff
@@ -1959,7 +1944,25 @@ pub mod pallet {
 				assert_matches!(
 					take_events().as_slice(),
 					[
+						TestEvent::Uniques(pallet_uniques::Event::Created {
+							collection: 0,
+							creator: 12362995669352346739,
+							owner: 12362995669352346739
```

Comment:
```suggestion
							collection: 0,
							creator: _,
							owner: _
```

Don't need to put the actual underlying `u64` here.

Reply:
done